### DISCUSSION TITLE
Newbie-learning-python/python/%E7%AC%AC1%E7%AB%A0%EF%BC%9APython%20%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86/1.%20Python%20%E7%9A%84%E8%B5%B7%E6%BA%90%E4%B8%8E%E5%8F%91%E5%B1%95

### DISCUSSION THREAD

**Discussion by giscus** [xiaobaiTech/golang-full-stack#2] [Announcements]
# Newbie-learning-python/python/%E7%AC%AC1%E7%AB%A0%EF%BC%9APython%20%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86/1.%20Python%20%E7%9A%84%E8%B5%B7%E6%BA%90%E4%B8%8E%E5%8F%91%E5%B1%95

Python 的诞生 Python 的创始人 Guido van Rossum 于 1989 年开始了 Python 的开发工作。当时，他在荷兰的 CWI（Centrum Wiskunde & Informatica）工作，目的是创建一门简单易读、功能强大的编程语言。 Guido van Rossum 设计 Python 的初衷是为了解决其他编程语言的...

https://xiaobaitech.github.io/Newbie-learning-python/python/%E7%AC%AC1%E7%AB%A0%EF%BC%9APython%20%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86/1.%20Python%20%E7%9A%84%E8%B5%B7%E6%BA%90%E4%B8%8E%E5%8F%91%E5%B1%95.html

**Comment by xiaobaiTech** [comment]
第一篇文章，学习打开从这里开始！
